cmd/util/flags/configflags/ipfs.go

```diff
@@ -4,12 +4,19 @@ import "github.com/bacalhau-project/bacalhau/pkg/config/types"
 
 var IPFSFlags = []Definition{
 	{
-		FlagName:             "ipfs-swarm-addr",
+		FlagName:             "ipfs-swarm-addrs",
 		ConfigPath:           types.NodeIPFSSwarmAddresses,
 		DefaultValue:         Default.Node.IPFS.SwarmAddresses,
 		Description:          "IPFS multiaddress to connect the in-process IPFS node to - cannot be used with --ipfs-connect.",
 		EnvironmentVariables: []string{"BACALHAU_IPFS_SWARM_ADDRESSES"},
 	},
+	{
+		FlagName:             "ipfs-swarm-key",
+		ConfigPath:           types.NodeIPFSSwarmKeyPath,
+		DefaultValue:         Default.Node.IPFS.SwarmKeyPath,
+		Description:          "Optional IPFS swarm key required to connect to a private IPFS swarm",
+		EnvironmentVariables: []string{"BACALHAU_IPFS_SWARM_KEY"},
```

Comment:
This value is optional, right now this setup results in 2 environment variables for this value.
1. `BACALHAU_NODE_IPFS_SWARMKEYPATH` which is derived from the `ConfigPath`
2. the one defined here.

Since this is a new env var would could drop this line and just use the automatic env var from 1. What do you think? The one you've defined is a lot easier on the eyes.

Reply:
Yeah I do think the manually defined one is more "natural". I think I'm in favour of just having both. An automatic one is still useful for situations where we need to programmatically define env vars.